x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/inference/loadingservice/ModelLoadingService.java

```diff
@@ -745,14 +750,13 @@ private void cacheEvictionListener(RemovalNotification<String, ModelAndConsumer>
 
     @Override
     public void clusterChanged(ClusterChangedEvent event) {
-        final boolean prefetchModels = event.state().nodes().getLocalNode().isIngestNode();
-        // If we are not prefetching models and there were no model alias changes, don't bother handling the changes
-        if ((prefetchModels == false)
```

Comment:
Is this changing the behaviour beyond what the release note says?

Previously in the case where an ingest pipeline referenced a model that didn't exist, that model would be prefetched on ingest nodes as soon as it was created. But now that doesn't happen? Or am I misreading this change?

Maybe the old behaviour was unacceptably resource-intensive, because it was doing a lot of processing on ingest nodes on every cluster state update, and changing it is justified. But in that case I think the release note should say this rather than pretend the change is only about logging.

Reply:
I'll make those changes in another PR